commitserver/commit/secure_mkdir_default.go

```diff
@@ -0,0 +1,22 @@
+//go:build !linux
+
+package commit
+
+import (
+	"fmt"
+	"os"
+
+	securejoin "github.com/cyphar/filepath-securejoin"
+)
+
+func SecureMkdirAll(root, unsafePath string, mode os.FileMode) (string, error) {
```

Comment:
It seems an useful function that can be used in various contexts. Have you considered declaring it in `util/io/files` package?

Reply:
also, please add godoc

Reply:
Yep, good idea to move it. I looked around for other use cases and haven't seen any yet. But util/io/files is a better home.